connect_four.py

```diff
@@ -1,8 +1,12 @@
 from enum import Enum
 
 class ColorEnum(Enum):
-        Yellow = "Y"
-        Red = "R"
+    Yellow = "Y"
+    Red = "R"
+
+class ErrorCodes(Enum):
```

Comment:
suggestion: have this include not just error codes but also status code where a valid move represents the value 0.

Reply:
enums are singular by convention, so it would ErrorCode